bundles/org.openhab.binding.govee/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="govee"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="govee-light">
+		<label>Govee Light Thing</label>
+		<description>Govee light controllable via LAN API</description>
+
+		<channels>
+			<channel id="brightness" typeId="system.brightness"/>
+			<channel id="color" typeId="system.color"/>
+			<channel id="color-temperature-abs" typeId="colortemperatureabs"/>
```

Comment:
Perhaps there should also be a `color-temperature` channel? See for example https://www.openhab.org/addons/bindings/hue/doc/readme_v2.html#channels-for-devices

Reply:
Channel has been added as well.